### PR TITLE
Add example for results in global coordinates

### LINKED ISSUE #434: Add documentation for rotate_to_global
The backend now supports the rotation of the stresses to global coordinates. We should mention this in an example and in the docs.

### PR DESCRIPTION
**Pull request by janvonrickenbach**
closes #434

### CODE REVIEW

**Review by greschd — APPROVED** [review]

**Comment by greschd** [comment]
LGTM, I only wonder if this is prominent enough, or if it should be a separate example for increased visibility.

**Comment by janvonrickenbach** [comment]
> LGTM, I only wonder if this is prominent enough, or if it should be a separate example for increased visibility.

I would say this is more of niche use case and fits well with the filtering examples (it does not make much sense to analyse data in global coordinates without filtering by ply). So I would keep it here.

File: examples/006_filter_composite_data_example.py [hunk 1]
```diff
@@ -99,16 +101,37 @@
...
         spot_reduction_strategy=SpotReductionStrategy.MAX,
         requested_location=dpf.locations.elemental,
     )
 
-    composite_model.get_mesh().plot(elemental_max)
+    composite_model.get_mesh().plot(elemental_values)
+
+# %%
+# The results can also be requested in global coordinates. This example gets the stress values in
+# the global coordinate system, selects the top spot of a selected ply and averages the values
```
Comment (PipKat) [inline examples/006_filter_composite_data_example.py:117]:
```suggestion
# the global coordinate system, selects the top spot of a selected ply, and averages the values
```
We use Oxford commas, which is merely a comma before the "and" or the "or" in a list of three or more items.

File: examples/006_filter_composite_data_example.py [hunk 2]
```diff
@@ -78,7 +78,9 @@
 # Get the stress field.
 stress_operator = composite_model.core_model.results.stress()
 stress_operator.inputs.bool_rotate_to_global(False)
-stress_field = stress_operator.get_output(pin=0, output_type=dpf.types.fields_container)[0]
+stress_field_material_coord = stress_operator.get_output(
```
Comment (roosre) [inline examples/006_filter_composite_data_example.py:81]: Should we mention what happens if this option is not set? What is the default?

**Review by roosre — APPROVED** [review]

File: examples/006_filter_composite_data_example.py (hunk 2 above)
Comment (janvonrickenbach) [reply examples/006_filter_composite_data_example.py:81]: Good point. The default is true. I will add a comment

**Pull request merged**